ui/desktop/app/routes/application.js

```diff
@@ -31,10 +31,14 @@ export default class ApplicationRoute extends Route {
   }
 
   /**
-   * Add OS config on controller.
+   * Add window frame config on controller.
    */
   async setupController(controller) {
-    controller.set('isMacOS', await this.ipc.invoke('isMacOS'));
+    controller.set('hasMacOSChrome', await this.ipc.invoke('hasMacOSChrome'));
```

Comment:
I like that these are performed once and statically set on the controller.  Initially I questioned this but since these values don't change at runtime, this is less costly.